src/app/shared/page/header/header.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, HostListener, NgZone } from '@angular/core';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styles: [
+    `
+      @media (min-width: 768px) {
+        .mt-3 {
+          margin-top: 0 !important;
+        }
+      }
+    `,
+  ],
```

Comment:
again

we can use classes `mt-3 md:mt-0`

Reply:
I add comment for that. same problem> I think it's conflict from Bootstrap
`<!-- Edit mt-3 class in ts file becucse it do conflict on the css code-->`